### PR TITLE
fix: styling issues with alert

### PR DESCRIPTION
**Pull request by nmerget**

### CODE REVIEW

File: packages/components/src/components/alert/alert.scss [hunk 1]
```diff
@@ -74,24 +77,23 @@
...
-			--icon-font-family: var(--db-base-icon-font-family);
-			--icon-font-size: var(--db-base-icon-font-size);
-		}
+	&::before {
+		/* TODO: Take filled icons if they are ready */
+		--icon-font-family: var(--db-base-icon-font-family);
+		--icon-font-size: var(--db-base-icon-font-size);
+		grid-area: icon;
+		margin-inline-end: 0;
 	}
```
Comment (annsch) [inline packages/components/src/components/alert/alert.scss:97]: @nmerget  doesn't this lead to problems in angular due to angulars component structure?

File: __snapshots__/alert/component/webkit/DBAlert-should-match-screenshot-for-variant-successful.png
Comment (annsch) [inline __snapshots__/alert/component/webkit/DBAlert-should-match-screenshot-for-variant-successful.png:1]: do we need an issue for this safari icon problem?

File: __snapshots__/alert/showcase/mobile-safari/regular/neutral-0/DBAlert-should-match-screenshot.png
Comment (annsch) [inline __snapshots__/alert/showcase/mobile-safari/regular/neutral-0/DBAlert-should-match-screenshot.png:1]: here as well, do we need or do we have an issue for the missing / false icon?

**Review by annsch — APPROVED** [review]

File: __snapshots__/alert/component/webkit/DBAlert-should-match-screenshot-for-variant-successful.png
Comment (nmerget) [reply __snapshots__/alert/component/webkit/DBAlert-should-match-screenshot-for-variant-successful.png:1]: @mfranzke said that it is just a problem with playwright shipping another webkit version. It works with the normal safari browser. I can't check it #windows

**Pull request merged**